src/sentry/api/serializers/models/group.py

```diff
@@ -694,27 +695,35 @@ def serialize(self, obj, attrs, user):
         return result
 
 
+SKIP_SNUBA_FIELDS = {
+    "status",
+    "bookmarked_by",
+    "assigned_to",
+    "for_review",
+    "assigned_or_suggested",
+    "unassigned",
+    "linked",
+    "subscribed_by",
+    "first_release",
+    "first_seen",
+}
+SKIP_SNUBA_FIELDS.update(SEARCH_TERMS)
+
+
 class GroupSerializerSnuba(GroupSerializerBase):
-    skip_snuba_fields = {
-        "status",
-        "bookmarked_by",
-        "assigned_to",
-        "for_review",
-        "assigned_or_suggested",
-        "unassigned",
-        "linked",
-        "subscribed_by",
-        "first_release",
-        "first_seen",
-        "last_seen",
-        "times_seen",
-        "date",  # We merge this with start/end, so don't want to include it as its own
-        # condition
-        # We don't need to filter by release stage again here since we're
-        # filtering to specific groups. Saves us making a second query to
-        # postgres for no reason
-        RELEASE_STAGE_ALIAS,
-    }
+    skip_snuba_fields = SKIP_SNUBA_FIELDS
+    skip_snuba_fields.update(
+        [
+            "last_seen",
+            "times_seen",
+            "date",  # We merge this with start/end, so don't want to include it as its own
+            # condition
+            # We don't need to filter by release stage again here since we're
+            # filtering to specific groups. Saves us making a second query to
+            # postgres for no reason
+            RELEASE_STAGE_ALIAS,
+        ]
+    )
```

Comment:
This will actually mutate `SKIP_SNUBA_FIELDS`, and so these extra fields will exist in both places this is used.

Instead of `.update`, we can copy like `skip_snuba_fields = {*SKIP_SNUBA_FIELDS, "last_see", ...}`